# Create a virtual environment where to download orthofinder

# Activate it, install: orthofinder, gget, genomepy

# Download the genomes using gget or genomepy
# Dmel, Drer,  Hsap,  Mmus,  Pri,  Smed,  Xtro,

# Download the TF database (perhaps pre-bundled?)

# Make sure the gene IDs match those of the TFDB

# Run orthofinder

# Retrieve Orthogroups.tsv

# Run a R script that expands the Orthogroups.tsv file into speciesX \t species from other animals, merge with TFDB, keep the ones that have match in TFDB, collapse the orthologs, and also the TFclasses

